### DISCUSSION TITLE
IRIS Shaderpack Development

### DISCUSSION THREAD

**Discussion by OttCS** [IrisShaders/Iris#1108] [Q&A]
The Continuum Shaders team used to have a great YouTube tutorial for writing a stable Optifine shader, but it was taken down a while ago due to being out of date.

I'm wondering if a tutorial for writing a very basic IRIS Shaderpack would be useful in garnering more community support. If not a tutorial, a very basic "default" shaderpack would be extremely helpful for developers.

On the topic of developing a shaderpack, I am currently working on the [GravityShade Shaderpack](https://github.com/OttCS/GravityShade) and I'm looking for some help with feature development and beta testers! I'd really appreciate it if anyone would check it out and give a feature request/bug report. Also, if anybody has experience developing shaders, any assistance would be greatly appreciated!

**Comment by BeansBeefBroccoli** [comment]
Somebody might do this at some point, but I think the devs gonna wait till the mod is more stable to make something like this.
